test/steps/mappers/create_kogito_service.go

```diff
@@ -167,6 +168,9 @@ func mapKogitoServiceConfigTableRow(row *TableRow, kogitoService *bddtypes.Kogit
 	case kogitoServiceInfinispanEnableEventsKey:
 		kogitoService.EnableEvents = MustParseEnabledDisabled(getThirdColumn(row))
 
+	case kogitoServiceExternalURIKey:
+		kogitoService.KogitoService.GetStatus().SetExternalURI(getThirdColumn(row))
```

Comment:
Hi @sutaakar , I found out that this code does not really set the status/externalURI, so the kogito runtime env variable `kogito.service.url` is not populated and by consequence the test fails. 
Do you have any tip to create a deployment like the following yaml?
```
apiVersion: app.kiegroup.org/v1alpha1
kind: KogitoRuntime
metadata:
  name: dmn-tracing-quarkus
spec:
  replicas: 1
  image:
    domain: quay.io
    namespace: jrota
    name: dmn-tracing-quarkus
    tag: 1.0.6-snapshot
    #  selector:
    # app: dmn-tracing-quarkus 
  kafka:
    externalURI: external-kafka-kafka-bootstrap:9092
status:
  externalURI: "http://dmn-tracing-quarkus:8080"
  conditions:
    - lastTransitionTime: "1985-04-12T23:20:50.52Z"
      lastUpdateTime: "1985-04-12T23:20:50.52Z"
      message: ""
      reason: ""
      status: ""
      type: ""
```

Reply:
Hi @r00ta IMHO we shouldn't set status fields, as this fields are used by Kogito operator to reflect current component status.
Could be `kogito.service.url` set as a runtime env variable directly in KogitoRuntime specs?

Reply:
Allright.. I found out that portion of is ignored by the cli (which is used to install the services during the tests)..

Reply:
Hi @sutaakar , I found out that the operator is anyway overwriting that env variable with the value in `kogitoService.KogitoService.GetStatus().GetExternalURI()`, I'm going to modify that code so that the operator sets that value if and only if `kogitoService.KogitoService.GetStatus().GetExternalURI() != ""`